src/store/global/global.actions.js

```diff
@@ -38,7 +38,13 @@ export const globalActionTypes = {
   CHECK_PENDING_DEPOSITS_SUCCESS: '[GLOBAL] CHECK PENDING DEPOSITS SUCCESS',
   LOAD_COORDINATOR_STATE: '[GLOBAL] LOAD COORDINATOR STATE',
   LOAD_COORDINATOR_STATE_SUCCESS: '[GLOBAL] LOAD COORDINATOR STATE SUCCESS',
-  LOAD_COORDINATOR_STATE_FAILURE: '[GLOBAL] LOAD COORDINATOR STATE FAILURE'
+  LOAD_COORDINATOR_STATE_FAILURE: '[GLOBAL] LOAD COORDINATOR STATE FAILURE',
+  LOAD_ESTIMATED_REWARD: '[GLOBAL] LOAD ESTIMATED REWARD',
```

Comment:
I would add the `AIRDROP` word to all the actions and callbacks, so it's more clear what they are referring to, for example: `LOAD_AIRDROP_ESTIMATED_REWARD`. If it takes you too much time we can leave it as it is, just a suggestion :)

Reply:
Technically it's not an airdrop, it's a reward system haha

Reply:
That's also true hahaha So I'll skip this part then, because if not maybe it's too redundant